core-bundle/assets/scripts/core.js

```diff
@@ -1675,6 +1799,26 @@ window.Backend =
 		});
 	},
 
+	/**
+	 * Update the "edit module" links in the module wizard
+	 *
+	 * @param {object} el The DOM element
+	 */
+	updateModuleLink: function(el) {
+		var td = el.getParent('tr').getLast('td'),
+			a = td.getElement('a.module_link');
+
+		a.href = a.href.replace(/id=[0-9]+/, 'id=' + el.value);
+
+		if (el.value > 0) {
+			td.getElement('a.module_link').setStyle('display', null);
+			td.getElement('img.module_image').setStyle('display', 'none');
+		} else {
+			td.getElement('a.module_link').setStyle('display', 'none');
+			td.getElement('img.module_image').setStyle('display', null);
+		}
```

Comment:
This will not work, as the `module_image` class does not exist anymore. See https://github.com/contao/contao/issues/5684#issuecomment-1386728717

Reply:
Fixed in 30f07c23763068bd1227c730a3e61d176fd547bb.